### DISCUSSION TITLE
Helm Chart for Splunk Operator

### DISCUSSION THREAD

**Discussion by li-bo-yang** [splunk/splunk-operator#815] [Ideas]
Hello everyone, my name is Boyang and I am a Product Manager Intern at Splunk. We are going to start working on a Helm chart for the Splunk Operator on Kubernetes, and I am wondering if everyone has any expectations and requirements.

We would appreciate it if you are able to provide us with some feedback as part of the development process. I will also post some questions we have in the thread and if you have an answer for it, feel free to comment.

If you are interested in testing out the Beta version in a couple of weeks or to have a chat with your comments and concerns, feel free to reach out to me at boyangl@splunk.com, and let's have a chat.

Thanks a lot!

**Comment by wzhiana** [comment]
i believe that one can be closed
